build-conf/zCEE3.properties

```diff
@@ -11,10 +11,21 @@ zcee3_shellEnvironment=bash
 #
 # Absolute path to gradle executable on z/OS UNIX System Services
 # for instance: /var/gradle/gradle-7.6/bin/gradle
-zcee3_gradlePath=
+zcee3_gradlePath=/var/gradle/gradle-8.10/bin/gradle
 
 #
-# JAVA Options used with gradle
-# for instance: zCEE_gradle_JAVA_OPTS=-Xmx512m -XX:MaxMetaspaceSize=256m -Dfile.encoding=UTF-8
-zcee3_gradle_JAVA_OPTS=-Dfile.encoding=UTF-8
+# JVM Options used with gradle
+# The use of IPv6 should be currently disabled for gradle and ava 11 on z/OS
```

Comment:
I don't see `zcee3_gradle_IBM_JAVA_OPTIONS` being used in the zCEE3.groovy language script. Can you please explain?

Reply:
Yes it's deprecated. I should completely remove this reference.
The only way would be the .profile approach.